docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+
+![](media/getting-started-vscode/vscode-fsi.png)
+
+This did three things:
+
+1. It started the FSI process.
+2. It sent the code you highlighted over the FSI process.
+3. The FSI process evaluated the code you sent over.
+
+Because what you sent over was a [function](../../language-reference/functions/index.md), you can now call that function with FSI!  In the interactive window, type the following:
+
+```fsharp
+toPigLatin "banana";;
+```
+
+You should see the following result:
+
+```fsharp
+val it : string = "ananabay"
+```
+
+Now, let's try with a vowel as the first letter. Enter the following:
+
+```fsharp
+toPigLatin "apple";;
+```
+
+You should see the following result:
+
+```fsharp
+val it : string = "appleyay"
+```
+
+The function appears to be working as expected.  Congratulations, you just wrote your first F# function in Visual Studio Code and evaluated it with FSI!
+
+>[!NOTE]
+As you may have noticed, the lines in FSI are terminated with `;;`.  This is because FSI allows you to enter multiple lines.  The `;;` at the end lets FSI know when the code is finished.
+
+## Explaining the code
+
+If you're not sure about what the code is actually doing, here's a step-by-step.
+
+As you can see, `toPigLatin` is a function which takes a word and converts it to a Pig-Latin representation of that word.  The rules for this are as follows:
+
+If the first character in a word starts with a vowel, add "yay" to the end of the word.  If it doesn't start with a vowel, move that first character to the end of the word and add "ay" to it.
+
+You may have noticed the following in FSI:
+
+```
+val toPigLatin : word:string -> string
+```
+
+This states that `toPigLatin` is a function which takes in a `string` as input (called `word`), and produces another `string`.  This is known as the [type signature of the function](https://fsharpforfunandprofit.com/posts/function-signatures/), a fundamental piece of F# that's key to understanding F# code.  You'll also notice this if you hover over the function in Visual Studio Code.
+
+In the body of the function, you'll notice two distinct parts:
+
+1. An inner function, called `isVowel`, which determines if a given character (`c`) is a vowel by checking if it matches one of the provided patterns via [Pattern Matching](../../language-reference/pattern-matching.md):
+
+[!code-fsharp[ToPigLatin](../../../../samples/snippets/fsharp/getting-started/to-pig-latin.fsx#L2-L6)]
+
+3. An [`if..then..else`](../../language-reference/conditional-expressions-if-then-else.md) expression which checks if the first character is a vowel, and constructs a return value out of the input characters based on if the first character was a vowel or not:
+
+[!code-fsharp[ToPigLatin](../../../../samples/snippets/fsharp/getting-started/to-pig-latin.fsx#L8-L11)]
```

Comment:
This list is not formatted correctly, the code samples should be indented by at least one space, so that Markdown understands they are part of the list.

Also, I would change `3.` to `2.`, but that should not have any effect on the output.